Sources/Props/ProgressView/BarsProgressStyle.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  BarsProgressStyle.swift
+//  
+//
+//  Created by Djuro on 1/24/22.
+//
+
+import SwiftUI
+
+public struct BarsProgressStyle: ProgressViewStyle {
+    
+    // MARK: - Initializer
+    public init() {}
+    
+    // MARK: - ProgressViewStyle
+    public func makeBody(configuration: Configuration) -> some View {
+        IndeterminateView(count: 8,
+                          spacing: 8,
+                          cornerRadius: 8,
+                          scaleRange: 0.5...1,
+                          opacityRange: 0.25...1)
+            .frame(width: 200, height: 50)
```

Comment:
You're obsessed with this frame size! 😂 It looks fine here, but it was really huge for the toggles which wouldn't make them compatible with system toggles, so that's why I changed those. I feel like it's weird to force that size here, although Apple does that with their spinner, so there's some established approach there. But in SwiftUI it's nice to think about Views letting themselves figure their frames out. It's also nice as a developer not having to worry about setting every frame. Not sure where I'm going. Maybe a maxWidth, maxHeight instead?

Reply:
Yes, you are right, that should be `maxWidth` and `maxHeight` 😆